pkgs/by-name/ac/acpica-tools/package.nix

```diff
@@ -46,11 +51,20 @@ stdenv.mkDerivation rec {
 
   installFlags = [ "PREFIX=${placeholder "out"}" ];
 
-  meta = with lib; {
+  passthru.updateScript = nix-update-script { };
```

Comment:
I'm not opposed to it, but I don't think it's related to nixfmt. Please move that to a separate commit.